specs/phase0/beacon-chain.md

```diff
@@ -388,33 +384,28 @@ class DepositData(Container):
     signature: BLSSignature  # Signing over DepositMessage
 ```
 
-#### `BeaconBlockHeader`
-
-```python
-class BeaconBlockHeader(Container):
-    slot: Slot
-    proposer_index: ValidatorIndex
-    parent_root: Root
-    state_root: Root
-    body_root: Root
-```
-
-#### `SigningRoot`
+#### `SigningData`
 
 ```python
-class SigningRoot(Container):
+class SigningData(Container):
```

Comment:
PR description mentions:
> Introduce `uniqueness_root` in signing root

Reply:
The `uniqueness_root` gets incorporated in the signing root (the root of the signing data).